### DISCUSSION TITLE
List file?

### DISCUSSION THREAD

**Discussion by snowboyz0825** [TurboWarp/extensions#839] [Ideas]
Could you add a simple text file with the links of all the extensions files within, I like to have all extensions loaded into Turbowarp at once. I would pull the text file, put it in a list, and use shovelutils to load all the extensions in a sprite and then have it delete itself. I'd import the sprite, have it do it's thing, and it'd just be a nice QoL feature.

**Comment by snowboyz0825** [comment]
It would have to be the turbowarp link as I don't think there is an extension that would allow me to get the file from github.

**Reply by yuri-kiss** [reply]
use cors proxy

**Reply by yuri-kiss** [reply]
heres a example of what your trying to do
![image](https://github.com/TurboWarp/extensions/assets/135030944/288ce972-5123-4990-aa51-dfd123a37532)

**Reply by LilyMakesThings** [reply]
the design is very human

**Reply by yuri-kiss** [reply]
lol wdym

**Reply by yuri-kiss** [reply]
this is max zoom-out
and this is all the extensions loaded
![image](https://github.com/TurboWarp/extensions/assets/135030944/b8d8cde3-956a-40dd-a985-ead5317928ee)

**Reply by yuri-kiss** [reply]
better version:
![image](https://github.com/TurboWarp/extensions/assets/135030944/ba8f3ed9-768c-4657-89a7-e34bed561507)

**Comment by yuri-kiss** [comment]
ok I made something that will import every extension without needing a new file on the site (github does not support sprite3 😭)
https://mega.nz/file/kid3VZTA#P_B1yEUeI5aEkzUQlFY3E3xc0UjELfWsjGHQIS4A28s

**Comment by snowboyz0825** [comment]
> ok I made something that will import every extension without needing a new file on the site (github does not support sprite3 😭) https://mega.nz/file/kid3VZTA#P_B1yEUeI5aEkzUQlFY3E3xc0UjELfWsjGHQIS4A28s

With a bit more technical knowhow, I now know that to load all the unsandboxed extensions at once wouldn't be possible using conventional blocks. I'm glad to have this to load all sandboxed extensions, but I'm going to close this now
